src/Layout/Header/Tools/index.tsx

```diff
@@ -58,8 +75,123 @@ export default class HeaderTools extends React.PureComponent<Props, State> {
     this.props.changeTheme(theme);
   }
 
+  private showAlert(alert: AlertItem): void {
+    this.appAlerts.showAlert(alert);
+  }
+
+  private get host(): string {
```

Comment:
I think that class method fits better here than getter:

```suggestion
  private getHost(): string {
```

Reply:
Let it be ```getHost```.